doc/tagbar.txt

```diff
@@ -123,10 +127,10 @@ The following requirements have to be met in order to be able to use tagbar:
 
   - Vim 7.0 or higher. Older versions will not work since Tagbar uses data
     structures that were only introduced in Vim 7.
-  - Exuberant ctags 5.5 or higher. Ctags is the program that generates the
-    tag information that Tagbar uses. It is shipped with most Linux
-    distributions, otherwise it can be downloaded from the following
-    website:
+  - Exuberant ctags 5.5 or higher, or universal-ctags. Ctags is the program 
+    that generates the tag information that Tagbar uses. It is shipped with
+    most Linux distributions, otherwise it can be downloaded from the 
+    following website:
 
         http://ctags.sourceforge.net/
```

Comment:
Lets go farther with this like we already did in the README to push for UCtags over ECtags.

```suggestion
  - At a minimum Exuberant Ctags >= 5.5, or (highly recommended) any version
    of Universal Ctags which is a currently maintained fork of Exuberant Ctags
    with many bugfixes, support for many more formats, and proper Unicode
    support. Some additional formats can also be handled by other providers
    such as jsctags, phpctags, or others.

    Universal Ctags cat be downloaded from https://ctags.io/
```